### DISCUSSION TITLE
Add a group margin, padding class

### DISCUSSION THREAD

**Discussion by anuraglol** [tailwindlabs/tailwindcss#5291] [Ideas]
Hey Developers,
I wanna explain this, so when I want to apply margin or padding to a bunch of related elements, let's say for all the list items, I wanna apply m-2 class to it, so there should be a better way to do this instead of adding the same class to every element.

**Comment by dance2die** [comment]
You can apply a base(global) CSS rule in your tailwind css file
**Doc**: https://tailwindcss.com/docs/adding-base-styles
**Demo**: https://play.tailwindcss.com/iCYq251Hfd

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  ul li {
    @apply m-2;
  }
}
```

Doing so would apply `m-2` to all `ul li` elements.  
And you can also override it in your HTML.

```html
<li class="flex items-start m-16">
```
, which will apply `m-16` to the particular `li` element.

**Reply by anuraglol** [reply]
Thank you for your help!!

**Comment by aggreggator** [comment]
Take a look at the documentation: https://tailwindcss.com/docs/space

**Reply by anuraglol** [reply]
Alright.. Thanks
